app/assets/stylesheets/navigation.scss

```diff
@@ -29,12 +32,35 @@
 }
 #search_bar {
   width: 115px;
+  border-top: 0px;
+  border-right: 0px;
+  border-left: 0px;
+  box-shadow: none;
+  border-color: #ffffff;
+}
+
+#search_bar:hover {
+  border-color: $neutral;
+}
+
+#search_bar:focus {
+  border-color: $neutral;
 }
 
 #search_button {
   width: 95px;
 }
 
+#search_button > button {
+  background-color: $neutral;
+  color: #777;
```

Comment:
Color literals like `#777` should only be used in variable declarations; they should be referred to via variable everywhere else.